src/bases.jl

```diff
@@ -17,10 +17,10 @@ Every vector space `fiber` is supposed to provide:
 Optionally:
 * inner product via `inner` (used to provide Riemannian metric on vector
   bundles),
-* [`flat`](@ref Main.Manifolds.flat) and [`sharp`](@ref Main.Manifolds.sharp),
+* `flat` and `sharp`,
```

Comment:
This is a somewhat unfortunate consequence of splitting documentation. Well, we could at least have an external link to docs of Manifolds.jl: https://juliamanifolds.github.io/Manifolds.jl/stable/features/atlases.html#Manifolds.flat-Tuple{AbstractManifold,%20Any,%20Any}

Reply:
I put some external links as well when fixing most of the docs, - sure we can link flat as well.